lighthouse-core/audits/byte-efficiency/unused-css-rules.js

```diff
@@ -140,7 +140,7 @@ class UnusedCSSRules extends Audit {
       const contentPreview = UnusedCSSRules.determineContentPreview(stylesheetInfo.content);
       url = '*inline*```' + contentPreview + '```';
     } else {
-      url = URL.getDisplayName(url);
+      url = URL.getDisplayName(url, {removeQuery: false});
```

Comment:
false as an overriding option is kinda awkward. (just like on the CLI)

how about `preserveQuery: true` so we can flip these.